Technical-Workshop/Presentation/Home Screen/View/HomeScreenViewController.swift

```diff
@@ -7,23 +7,49 @@
 
 import UIKit
 
-class HomeScreenViewController: UIViewController {
-
+class HomeScreenViewController: UIViewController, UICollectionViewDelegate, UICollectionViewDataSource, UICollectionViewDelegateFlowLayout, UITableViewDelegate, UITableViewDataSource {
+    
+    @IBOutlet weak var categoriesCollectionView: UICollectionView!
+    @IBOutlet weak var homeTableView: UITableView!
+    
+    
+    private lazy var viewModel: HomeScreenViewModelType = {
+        return HomeScreenViewModel()
+    }()
+        
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        // Do any additional setup after loading the view.
+    }
+ 
+    // MARK: Collection View Callback Methods
+    
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return viewModel.categoryItemsCount
```

Comment:
if the function is one line and returns at this first line you can omit the return keyword